components/tikv_util/src/sys/cgroup.rs

```diff
@@ -200,13 +202,33 @@ impl CGroup {
         Self { path }
     }
 
-    pub fn read_num(&self, param: &str) -> Result<i64, Box<dyn error::Error>> {
+    pub fn read_line(&self, param: &str) -> io::Result<String> {
```

Comment:
Does it need to be `pub`?

Reply:
It seems to be unnecessary, fixed in [08d6c9c](https://github.com/tikv/tikv/pull/9043/commits/08d6c9ca6fe4e3ed2b000e142b3c585ce1301783).